File: Creacion API/APIRest2_23_24_Capas/APIRest2_23_24/src/main/java/es/luisherrero/apirest1/model/Departamento.java
package es.luisherrero.apirest1.model;

import java.io.Serializable;

import jakarta.persistence.Column;
import jakarta.persistence.Id;
import jakarta.persistence.Table;

@jakarta.persistence.Entity
@Table(name="departamentos")
public class Departamento implements Serializable{
	private static final long serialVersionUID=1;
	@Id
	@Column(name="int")
	private int int_;

	private byte activo;

	private String cod;

	private String nombre;

	// En la clase Departamento
	@Column(name="jefedep_id")
	private int jefedep_id;

	public Departamento() {
	}

	public int getInt_() {
		return this.int_;
	}

	public void setInt_(int int_) {
		this.int_ = int_;
	}

	public byte getActivo() {
		return this.activo;
	}

	public void setActivo(byte activo) {
		this.activo = activo;
	}

	public String getCod() {
		return this.cod;
	}

	public void setCod(String cod) {
		this.cod = cod;
	}

	public String getNombre() {
		return this.nombre;
	}

	public void setNombre(String nombre) {
		this.nombre = nombre;
	}

	public int getPersonal() {
			return this.jefedep_id;
	}

	public void setPersonal(int personal) {
		this.jefedep_id = personal;
	}
}

File: Creacion API/Python e imagenes/departamento.py
class Departamento:
    def __init__(self, int, cod, nombre, activo, jefedep_id):
        self.int = int
        self.cod = cod
        self.nombre = nombre
        self.activo = activo
        self.jefedep_id = jefedep_id

    def get_int(self):
        return self.int

    def set_int(self, value):
        self.int = value
        
    def get_cod(self):
        return self.cod

    def set_cod(self, value):
        self.cod = value
    
    def get_nombre(self):
        return self.nombre

    def set_nombre(self, value):
        self.nombre = value
    
    def get_activo(self):
        return self.activo

    def set_activo(self, value):
        self.activo = value
        
    def get_jefedep_id(self):
        return self.jefedep_id

    def set_jefedep_id(self, value):
        self.jefedep_id = value